### PR TITLE
Create skeleton comp

### PR DESCRIPTION
**Pull request by KaiWaldrant**
This component creates skeleton files:

* config.vsh.yaml
* script.py/R

depending on the script language, task, and component.

### CODE REVIEW

File: src/common/create_skeleton/script.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+  
+  tmpl['functionality']['info']['metrics'] = [{
+      'metric_id': 'metric_id',
+      'metric_name': 'Metric Name',
+      'metric_description': 'metric description',
+      'min': 0,
+      'max': 1,
+      'maximize': 'true',
+    }
+    ]
```
Comment (rcannood) [inline src/common/create_skeleton/script.py:29]:
```suggestion
  tmpl['functionality']['info']['metrics'] = [{
      'metric_id': 'metric_id',
      'metric_name': 'Metric Name',
      'metric_description': 'metric description',
      'min': 0,
      'max': 1,
      'maximize': 'true',
    }]
```

File: src/common/create_skeleton/script.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+  return conf
+
+def add_r_setup(conf):
+
+  conf['functionality']['resources'][0]['type'] = 'r_script'
+  conf['functionality']['resources'][0]['path'] = 'script.r'
+
+  conf['functionality']['test_resources'][0]['type'] = 'r_script'
+  conf['functionality']['test_resources'][0]['path'] = 'script.r'
```
Comment (rcannood) [inline src/common/create_skeleton/script.py:68]:
```suggestion

  conf['functionality']['resources'][0]['type'] = 'r_script'
  conf['functionality']['resources'][0]['path'] = 'script.R'

  conf['functionality']['test_resources'][0]['type'] = 'r_script'
  conf['functionality']['test_resources'][0]['path'] = 'script.R'
```

**Review by rcannood — CHANGES_REQUESTED** [review]

**Comment by KaiWaldrant** [comment]
I'm thinking about creating 2 skeleton components for metric and methods. I still need to add an py/Rscript template for metrics and this is going to make the script too "complicated".

What do you think @rcannood ?

**Comment by rcannood** [comment]
Hmm.. I would prefer 1 component, I think. Am I right in understanding that it ATM it works as intended?

In the future, I might consider using a python script + jinja templating engine, but if it works we can leave it as is for now.

**Comment by KaiWaldrant** [comment]
yes this should work as intended

File: src/common/create_skeleton/script.py
Comment (KaiWaldrant) [inline src/common/create_skeleton/script.py:1]: as mentioned in the file comments the output anndata still contain `adata`

Comment (rcannood) [reply src/common/create_skeleton/script.py:1]: Phew.

**Pull request merged**